src/components/new-request/search.vue

```diff
@@ -860,6 +862,14 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
     this.setNoCorpNum(false)
   }
 
+  /** Resets fields when returned to the Tabs component */
+  @Watch('getDisplayedComponent')
+  watchDisplayedComponent (displayedComponent: string) {
+    if (displayedComponent === 'Tabs') {
+      this.onBusiness(null)
+    }
+  }
```

Comment:
This is switching between "Get a Business Name" and "Manage My Name Request", right?

Check also what the "Return" or "Reset" buttons do (once you're past the second or 3rd page). I think some things aren't reset there either.